src/node/stage/bootstrapping.rs

```diff
@@ -187,12 +180,18 @@ impl Bootstrapping {
             *relocate_details.destination(),
         );
 
-        let new_full_id = FullId::within_range(&mut core.rng, &name_prefix.range_inclusive());
-        let relocate_payload =
-            RelocatePayload::new(relocate_details, new_full_id.public_id(), &core.full_id)?;
+        // FIXME: do we need to reuse MainRng everywhere really??
```

Comment:
The rng reuse is there to maintain reproducible tests. But it only makes sense for tests that use the mock network. Because we seem to be moving away from it, we can maybe remove it too.